code/modules/xenoarcheaology/effects/cannibal.dm

```diff
@@ -0,0 +1,75 @@
+/datum/artifact_effect/cannibalfeeling
+	name = "cannibalfeeling"
```

Comment:
Very morbid but if you stab a dead (or perhaps even unconscious) body with a fork, do you get a bite of flesh? Or would you have to gibber them to get their tasty meats?

Reply:
Sadly, no, but you can chop their arm off and munch. Or their hand, if you're nice.